gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "mon_shoggoth",
+    "fg": [
+        { "weight": 1, "sprite": "mon_shoggoth" }
```

Comment:
why bother having a list if it's just one element? Just make it `"fg": "mon_shoggoth"`. And you can omit empty elements, no need for all these `[""]`.

Reply:
And if the `mon_shoggoth` is not used anywhere else you can just remove the JSON and rely on `compose.py --use-all`

Reply:
I'm not sure i understand. It's one sprite, not a list. 
I copy pasted your code and it seems all the conflicts are gone!👍

Reply:
i also deleted the json, compose py adds an image with the shoggoth but when i spawn one in it shows the fallback sprite not the shoggoth one

Reply:
i think it's probably the conflicts, now that i deleted that json they're back

Reply:
By list I meant a JSON Array. This is an array with one object:
```json
[
        { "weight": 1, "sprite": "mon_shoggoth" }
]
```
and this is an array with two objects
```json
[
        { "weight": 1, "sprite": "mon_shoggoth" },
        { "weight": 1, "sprite": "mon_shoggoth2" }
]
```